addOns/ascanrules/src/test/java/org/zaproxy/zap/extension/ascanrules/CommandInjectionScanRuleUnitTest.java

```diff
@@ -197,26 +194,6 @@ void shouldDefaultTo5SecsIfConfigTimeIsMalformedValueForTimeBasedAttacks() throw
         assertThat(rule.getTimeSleep(), is(equalTo(5)));
     }
 
-    @Test
```

Comment:
Tests shouldn't be removed I believe :)

Reply:
Thanks for the catch, the removal was an accident: the test passes fine.